tests/unit/api/v1/test_records.py

```diff
@@ -778,7 +779,7 @@ async def test_create_record_response_with_extra_question_responses(
                         "output_ok": {"value": False},
                     },
                 },
-                "Expected text value, found <class 'bool'>",
+                "Expected text value, found <class 'int'>",
```

Comment:
Why did this error detail change?

Reply:
Okay. So, this depends on the accepted value types defined for the schema. As a user, I this this message can be weird since the user never sends an integer. 

Maybe makes sense to implement some custom parsing in a future PR.

Reply:
We can discuss the use of strict types to avoid this kind of things.

Reply:
Nope. I prefer this to avoiding float values for rating responses, for example.